nunaserver/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   celery:
-    image: python:latest
+    image: python:3.9
```

Comment:
Remind me please, why is it not possible to use the latest Python here?

Reply:
Because Celery 5.1 is only supported up to Python 3.9.

Reply:
Can we switch to the latest celery?

Reply:
will test it

Reply:
works with the newest version of Celery